test/geometry/transform/test_imgwarp.py

```diff
@@ -268,6 +278,35 @@ def test_fill_padding_channels(self, device, dtype, num_channels):
 
         assert_close(img_a[:, :, :1, :1].squeeze(), fill_value.squeeze())
 
+    @pytest.mark.parametrize("align_corners", (True, False))
+    @pytest.mark.parametrize("padding_mode", ("zeros", "fill"))
+    def test_jit_script(self, device, dtype, align_corners, padding_mode):
+        offset = 1.0
+        h, w = 3, 4
+        aff_ab = torch.eye(2, 3, device=device, dtype=dtype)[None]
+        aff_ab[..., -1] += offset
+
+        img_b = torch.arange(float(3 * h * w), device=device, dtype=dtype).view(1, 3, h, w)
+        net = DummyNNModule(3, 4, align_corners, padding_mode)
+        script_net = torch.jit.script(net)
+
+        assert isinstance(script_net, torch.jit.ScriptModule)
+        assert_close(script_net(img_b, aff_ab), net(img_b, aff_ab))
+
+    @pytest.mark.parametrize("align_corners", (True, False))
+    @pytest.mark.parametrize("padding_mode", ("zeros", "fill"))
+    @pytest.mark.xfail(reason="aten::linalg_inv is not yet supported in ONNX opset version 14.")
```

Comment:
we can't add a condition to mark xfail just for this version?

Reply:
14 is the default option for torch 2.0.1, which is why I used it. The same is true for opset version 19 (most recent). I can change the comment to reflect that, or parametrize the test if you would like, but I don't think this test should pass in current state of things. So maybe it should be removed?

Reply:
as an idea, can we add something at the high level api, like a decorator to add in some sort of registry that includes min supported onnx version ?

Reply:
eventually to avoid this is issue around not supported `linalg_inv`:
- accept the inverted affine transform
- (or) implement closed solution of inversion for affine transform /cc @ducha-aiki

Reply:
if the test will not pass for the current versions, I would say to skip it instead of xfial

Reply:
done. I removed the test.